sktime/transformations/panel/shapelet_transform.py

```diff
@@ -999,6 +1000,9 @@ class RandomShapeletTransform(BaseTransformer):
     n_jobs : int, default=1
         The number of jobs to run in parallel for both `fit` and `transform`.
         ``-1`` means using all processors.
+    parallel_backend : str, default=None
+        Specify the parallelisation backend implementation in joblib, where
```

Comment:
what are valid values for this string?

Reply:
Ah yeah that is missing, will add it now.